scripts/ci/jobs/check-generated.sh

```diff
@@ -14,10 +14,14 @@ FAIL_FLAG="/tmp/fail"
 info 'Ensure that generated files are up to date. (If this fails, run `make proto-generated-srcs && make go-generated-srcs` and commit the result.)'
 function generated_files-are-up-to-date() {
     git ls-files --others --exclude-standard >/tmp/untracked
-    make proto-generated-srcs
     # Remove generated mocks, they should be regenerated and if source was deleted they should be deleted as well.
     git grep --files-with-matches "Package mocks is a generated GoMock package." -- '*.go' | xargs rm
+    # Remove generated comment, it will be regenerated and if source was changed comment should be removed as well.
```

Comment:
What about code generated by the migration bootstrap target?
```
// Code generated by make bootstrap_migration generator. DO NOT EDIT.
```

What about other code generation in migrations based on following the README to generate schemas and converters?  Should we flag those differently or should we exclude the migrator?

Reply:
Good point, let's skip migration for now.